plugin/vim-tmux-runner.vim

```diff
@@ -178,6 +178,19 @@ function! s:SendKeys(keys)
     call s:SendEnterSequence()
 endfunction
 
+function! s:SendKeysRaw(keys)
+  if !s:ValidRunnerPaneSet() | return | endif
+  call s:_SendKeys(a:keys)
+endfunction
+
+function! s:SendCtrlD()
+  call s:SendKeysRaw('')
+endfunction
+
+function! s:SendCtrlC()
+  call s:SendKeysRaw('')
```

Comment:
Well this is... odd. Looks like github doesn't display the control characters.

Reply:
Ha saw that. Was super confused until I opened it up in vim.